docs/packages/dara-components/graphs/introduction.mdx

```diff
@@ -226,7 +226,7 @@ config.add_page(
 
 #### `default_legends` Example
 
-Default legends is structured slightly differently. By default it takes a dict of key EditorMode and legends array value. This is so the correct default
+Default legends is structured slightly differently. By default it takes a dict of key [EditorMode](../../../../reference/dara/components/graphs/definitions#editormode) and legends array value. This is so the correct default
```

Comment:
you should be able to use ref links as `package.x.x.EditorMode`, no?

Reply:
I remember those were broken a while ago are they working again?

Reply:
they should be fixed, but you can also leave it as is if you are unsure 😊

Reply:
Will leave it now but later will test that it works for the docs on the website as well as internal :)